tests/github/test_github.py

```diff
@@ -384,8 +389,7 @@ def test_get_pages(
         )
     )
     assert mocked_api_call.call_count == 2
-    assert len(pr_rows) == 3
-    assert pr_rows[-1]['PR Name'] == 'chore(something): somethin'
```

Comment:
I'm not very sure why there was only 3 PRs displayed before. There is 3 sent in `extracted_prs_1` plus 1 in `extracted_prs_2`, so I think it's normal we change this to 4.

Reply:
ah ah this left me wondering where this 3 was coming from, thanks for resolving this issue